variants/backend-base/config/initializers/content_security_policy.rb

```diff
@@ -104,7 +104,7 @@
   # https://docs.sentry.io/error-reporting/security-policy-reporting/ for
   # details
   #
-  # policy.report_uri "https://o250825.ingest.sentry.io/api/1409600/security/?sentry_key=SOMETHING"
+  # policy.report_uri ENV.fetch("SENTRY_CSP_HEADER_REPORT_ENDPOINT")
```

Comment:
I think we might as well make this based on the variable being set:

```suggestion
  if ENV.has_key("SENTRY_CSP_HEADER_REPORT_ENDPOINT")
    policy.report_uri ENV.fetch("SENTRY_CSP_HEADER_REPORT_ENDPOINT")
  end
```

Based on the proceeding comment we might want to drop the `SENTRY_` prefix to make it generic, though I don't mind keeping it as an indicator that if you're using Sentry on a project you should be able to set this too